### PR TITLE
reuse deprecated IDs in the far future

### PR DESCRIPTION
**Pull request by robUx4**
They are listed in the IANA registry of known IDs.

### CODE REVIEW

**Comment by robUx4** [comment]
We lose valuable 0xFD, 0xEB,  0xE8,  0xCF,  0xCE,  0xCD,  0xCC,  0xCB,  0xCA,  0xC9,  0xC8,  0xC7,  0xC6,  0xC4,  0xC1,  0xC0,  0xAF,  0xAA,  0x97,  0x8E 1-bye IDs...

**Comment by robUx4** [comment]
Mark theses IDs as "returned" and tell IANA to use them last.

**Comment by robUx4** [comment]
As discussed in the meeting, I've marked them as "returned" although I could not find a proper definition for that. I added text about that in the IANA section and in the Annex A section.

**Review by mcr — APPROVED** [review]

File: iana_matroska_ids.md [hunk 1]
```diff
@@ -31,5 +31,9 @@ For example within a BlockGroup or Chapters. The Four-octet Element IDs are most
 They should only be used for such high level elements.
 Elements that are not expected to be used often should use Three-octet Element IDs.
 
+Elements found in (#annex-a-historic-deprecated-elements) have an assigned Matroska Element ID for historical reasons.
+These elements are not in use and **SHOULD NOT** be reused unless there is no other IDs available with the desired size.
+Such IDs are considered as `returned` to the IANA registry as they could be used for other things in the future.
```
Comment (mcr) [inline iana_matroska_ids.md:36]:
```suggestion
Such IDs are considered as `reclaimed` to the IANA registry as they could be used for other things in the future.
```

File: transforms/ebml_schema2markdown4iana_ids.xsl [hunk 2]
```diff
@@ -24,15 +24,18 @@
     <xsl:value-of select="@id"/>
     <xsl:text> | </xsl:text>
     <xsl:value-of select="@name"/>
-    <xsl:text> | Described in (#</xsl:text>
     <xsl:choose>
-      <xsl:when test="maxver='0'">
+      <xsl:when test="@maxver='0'">
+        <xsl:text> | Returned ((#</xsl:text>
```
Comment (robUx4) [inline transforms/ebml_schema2markdown4iana_ids.xsl:29]:
```suggestion
        <xsl:text> | Reclaimed ((#</xsl:text>
```

**Pull request merged**